client/signup/main.jsx

```diff
@@ -298,11 +301,23 @@ class Signup extends React.Component {
 		const flowSteps = flows.getFlow( flowName ).steps;
 		map( flowSteps, flowStepName => this.processFulfilledSteps( flowStepName, nextProps ) );
 
-		if ( includes( flows.excludedSteps, stepName ) ) {
+		if ( flows.excludedSteps.has( stepName ) ) {
 			this.goToNextStep( flowName );
 		}
 	};
 
+	addUnfulfilledSteps( nextProps ) {
```

Comment:
The logic in this method doesn't seem to match the name - I don't see anything being added anywhere?
Instead, it looks like it invokes call backs. Those callbacks themselves might add something somewhere but that's beyond the context of this particular unit of work.